jocd/src/br/org/certi/jocd/core/MemoryRangeBase.java

```diff
@@ -27,7 +27,12 @@
 
   public MemoryRangeBase(long start, long end, long length) {
```

Comment:
This method signature seems a bit weird. If the "lenght" param is passed, then "end" is useless?
I think it is simpler (and less confusing) if we just use start and lenght as parameters, and the end value is always calculated inside the method.

Reply:
Yes, I agree. But this is how pyOCD is implemented (and at least at this point we are following pyOCD "no matter what"). Should I change this?

Reply:
If it is a small refactor, if think it is worth the effort. Otherwise, we can address it later as an improvement.

Reply:
Fixed. 

Please note that I've update this pull request with commit 6457cf9 (Remove end address from MemoryRangeBase) and 780db21 (Fix integer to long conversions errors on enumerations).